release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -80,6 +80,12 @@ You can import and use images from any registry configured during or after the
 cluster installation by sharing the node's pull secret credentials with the
 `openshift-api`, `builder`, and `image-registry` Pods.
 
+[id="ocp-4-5-support-nodeaffinity-scheduler-in-image-registry-crd"]
+==== Support nodeAffinity scheduler in image registry CRD
+
+The nodeAffinity scheduler is now supported to ensure image registry
```

Comment:
Do you have to add the nodeAffinity scheduler to the image registry manually, or is it enabled by default? I'd say which here.

Reply:
It must be manually configured. @xiuwang do you by chance know the steps to configure nodeAffinity in the image registry custom resource?

Reply:
@codyhoag It's manually configured. We can use `oc patch config.image/cluster -p '{"spec":{"affinity":{"nodeAffinity":{"preferredDuringSchedulingIgnoredDuringExecution":[{"preference":{"matchExpressions":[{"key":"node-role.kubernetes.io/infra","operator":"Exists"}]},"weight":1}]}}}}' --type=merge` to add this scheduler to image registry CRD.

Reply:
@codyhoag We just need add summary about this feature not the details here, right?